SingularityUI/app/templates/requestDetail/requestActiveTasks.hbs

```diff
@@ -36,6 +36,9 @@
                     <td class="hidden-xs" data-value="{{ taskId.updatedAt }}">
                         {{timestampFromNow updatedAt}}
                     </td>
+                    <a href="{{ appRoot }}/task/{{ taskId.id }}/tail/{{ subtituteTaskId ../../config.runningTaskLogPath taskId.id }}" title="Log">
```

Comment:
oops, forgot to fix that when copying from the diffs- f219a603eb35e70fb8e9e0e6bb5a9efd990eca84